src/ReleaseHistory.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 
+* BUGFIX: Revert Json Serialization field order change and skip emit empty AutomationDetails node. [#2420](https://github.com/microsoft/sarif-sdk/pull/2420)
```

Comment:
>* BUGFIX: Revert Json Serialization field order change and skip emit empty AutomationDetails node. [](http://example.com/codeflow?start=0&length=98)

I was under the impression that the order change reversion was just until the other PR is approved and then we'll settle on a final order and consolidate the two changes.

Would this be better described accordingly:
"Standardizing Json Serialization field order and..."?

Reply:
I would consider this to be my only blocking comment at the moment.

Reply:
agree, let me change to word like "Adjust" instead of Standardizing I fear people think it is a whole solution standardizing by looking the title

Reply:
updated the title.